packages/gatsby-source-graphql/src/__tests__/gatsby-node.js

```diff
@@ -1,4 +1,4 @@
-jest.mock(`graphql-tools-fork`, () => {
+jest.mock(`graphql-tool`, () => {
```

Comment:
This seems to be breaking tests:

```suggestion
jest.mock(`graphql-tools`, () => {
```

Reply:
oops :)